reana_client/cli/workflow.py

```diff
@@ -156,6 +159,47 @@ def workflow_workflows(ctx, _filter, output_format, access_token,
             err=True)
 
 
+@click.command(
+    'sessions',
```

Comment:
In view of #253, perhaps call the command a `list`, which would by default do `list_workflows()`, and which would accept optional CLI argument `list --sessions` to list sessions.